templates/components/account/payment-methods-list.html

```diff
@@ -0,0 +1,46 @@
+{{/unless}}
+
+<ul class="paymentMethodList">
+    {{#each customer.payment_methods}}
+        <h4 class="paymentMethodName">{{provider.display_name}}</h4>
+        {{#each methods}}
+        <li class="paymentMethod">
+            <div class="panel panel--paymentMethod">
+                <div class="panel-body">
+                    <div class="paymentMethod-card">
+                        {{#if brand '===' 'american_express'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/american-express.svg'}}" alt="{{lang 'account.payment_methods.card_types.american_express'}}" title="{{lang 'account.payment_methods.card_types.american_express'}}">
+                            <span class="paymentMethod-brand">{{lang 'account.payment_methods.card_types.american_express'}} {{lang 'account.payment_methods.card_ending_in' last_four=last_4}}</span>
+                        {{/if}}
+                        {{#if brand '===' 'diners'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/diners.svg'}}" alt="{{lang 'account.payment_methods.card_types.diners'}}" title="{{lang 'account.payment_methods.card_types.diners'}}">
+                            <span class="paymentMethod-brand">{{lang 'account.payment_methods.card_types.diners'}} {{lang 'account.payment_methods.card_ending_in' last_four=last_4}}</span>
+                        {{/if}}
+                        {{#if brand '===' 'discover'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/discover.svg'}}" alt="{{lang 'account.payment_methods.card_types.discover'}}" title="{{lang 'account.payment_methods.card_types.discover'}}">
+                            <span class="paymentMethod-brand">{{lang 'account.payment_methods.card_types.discover'}} {{lang 'account.payment_methods.card_ending_in' last_four=last_4}}</span>
+                        {{/if}}
+                        {{#if brand '===' 'mastercard'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/mastercard.svg'}}" alt="{{lang 'account.payment_methods.card_types.mastercard'}}" title="{{lang 'account.payment_methods.card_types.mastercard'}}">
+                            <span class="paymentMethod-brand">{{lang 'account.payment_methods.card_types.mastercard'}} {{lang 'account.payment_methods.card_ending_in' last_four=last_4}}</span>
+                        {{/if}}
+                        {{#if brand '===' 'visa'}}
+                            <img class="paymentMethod-icon" src="{{cdn 'img/payment-methods/visa.svg'}}" alt="{{lang 'account.payment_methods.card_types.visa'}}" title="{{lang 'account.payment_methods.card_types.visa'}}">
+                            <span class="paymentMethod-brand">{{lang 'account.payment_methods.card_types.visa'}} {{lang 'account.payment_methods.card_ending_in' last_four=last_4}}</span>
+                        {{/if}}
```

Comment:
We actually store other credit card types (ie brands) in our vaults. Would you be able to add a fallback strategy for cards that do not match any of the brands listed here? 

I would suggest a generic credit card logo or no logo and copy that reads something like `Credit card ending in XXXX`.

🍹